docs/content/author-apps/networking/howto-tls-termination/snippets/tls-termination-data.bicep

```
@@ -0,0 +1,73 @@
+import radius as radius
+
+param environment string
+
+@description('Specifies TLS cert secret values.')
```

```suggestion
@description('TLS certificate data')
```

Does this need to be raw or base64 encoded? The description should specify.